imports/api/topics/topics.tests.js

```diff
@@ -172,7 +175,69 @@ if (Meteor.isServer) {
           chai.assert.equal(unseenComments[0].text, 'comment 1');
           chai.assert.equal(unseenComments[1].text, 'comment 2');
           chai.assert.equal(unseenComments[2].text, 'comment 4');   // 3 was deleted
-          chai.assert.equal(unseenComments[3].text, 'comment 5'); 
+          chai.assert.equal(unseenComments[3].text, 'comment 5');
+          done();
+        });
+      });
+
+      describe('statusChange', function () {
+        before(function () {
+          // Clear
+          Topics.remove({});
+          Comments.remove({});
+
+          // Create a ticket
+          topicId = Topics.methods.insert._execute({ userId: otherUserId }, {
+            communityId: Fixture.demoCommunityId,
+            userId: otherUserId,
+            category: 'ticket',
+            ticket: {
+              type: 'issue',
+              urgency: 'normal',
+            },
+            status: 'reported',
+            title: 'Just a ticket',
+            text: 'Not much to say',
+          });
+        });
+
+        it('doesn\'t let you change the status if you don\'t have the right permission', function (done) {
+          chai.assert.throws(() => {
```

Comment:
Ildi szolt hogy mostantol mindig irjuk ki hogy milyen exception-t varunk el hogy dobjon, mert kulonben ha valami egesz mas hibat dob, akkor a teszt ugy is zold lesz.

Reply:
Praktikusabb, ha a kódba írod: https://www.chaijs.com/api/assert/#method_throws
pl: chai.assert.throws(() => { do this thing }, 'err_permissionDenied');